packages/flutter/lib/src/foundation/consolidate_response.dart

```diff
@@ -3,31 +3,103 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
 import 'dart:io';
 import 'dart:typed_data';
 
-/// Efficiently converts the response body of an [HttpClientResponse] into a [Uint8List].
+/// Signature for getting notified when chunks of bytes are received while
+/// consolidating the bytes of an [HttpClientResponse] into a [Uint8List].
+///
+/// The `cumulative` parameter will contain the total number of bytes received
+/// thus far. If the response has been gzipped, this number will be the number
+/// of compressed bytes that have received _across the wire_.
```

Comment:
`number of (probably) compressed bytes`

Reply:
Why probably?  We're running it through a gzip decoded, so if it wasn't properly compressed, this method will throw.

Reply:
The transfer might not have been compressed at all.

Reply:
But I preface the statement with "If the response has been gzipped".  I don't think I'm understanding what you're saying.

Reply:
Oh I see. I misread it, sorry.